1.10/deploying-services/gpu.md

```diff
@@ -0,0 +1,175 @@
+1.  Install DC/OS using the [custom advanced installation instructions](/docs/1.10/installing/custom/advanced/).
+
+## AWS EC2 DC/OS Installation with GPUs
+
+###  Prerequisites
+- The AWS DC/OS advanced template [system requirements](/docs/1.10/installing/cloud/aws/advanced/).
+- The `zen.sh` script copied to your local machine. The script and instructions are [here](/docs/1.10/installing/cloud/aws/advanced/).
+
+### Create Dependencies
+
+1. Run the `zen.sh` script to create the Zen template dependencies. These dependencies will be used as input to create your stack in CloudFormation.
+  ```
+  bash ./zen.sh <stack-name>
+  ```
+  **Important:** You must run the `zen.sh` script before performing the next steps.
+
+1. Follow the instructions [here](/docs/1.10/installing/cloud/aws/advanced/) to create a cluster with advanced AWS templates, using the following GPU-specific configuration.
+
+1. On the **Create Stack** > **Specify Details** page, specify your stack information and click **Next**. Here are the GPU-specific settings.
+   
+   - **CustomAMI** - Specify the custom AMI for your region:
+
+      - us-west-2: `ami-9b5d97fb`
+      - us-east-1: `ami-e10e50f6`
+      - ap-southeast-2: `ami-37b28f54`
+  
+   - **MasterInstanceType** - Accept the default master instance type (e.g. `m3.xlarge`).
+   - **PrivateAgentInstanceType** - Specify an [AWS GPU machine type](https://aws.amazon.com/ec2/instance-types/#p2) (e.g., `g2.2xlarge`).
+   - **PublicAgentInstanceType** - Specify an [AWS GPU machine type](https://aws.amazon.com/ec2/instance-types/#p2) (e.g., `g2.2xlarge`).
+
+1. On the **Options** page, accept the defaults and click **Next**.
+
+   **Tip**: You can choose whether to rollback on failure. By default this option is set to **Yes**.
+
+1. On the **Review** page, check the acknowledgement box, then click **Create**.
+
+   **Tip**: If the **Create New Stack** page is shown, either AWS is still processing your request or you’re looking at a different region. Navigate to the correct region and refresh the page to see your stack.
+
+# Using GPUs in Your Apps
+
+You can specify GPUs in your application definitions with the `gpus` parameter.
+
+-  You can only specify whole numbers of GPUs in your application definition. If a fractional amount is selected, launching the task will result in a `TASK_ERROR`.
+-  NVIDIA GPU support is only available for tasks launched using the [DC/OS Universal container runtime](/docs/1.10/deploying-services/containerizers/). 
+-  You can set the `gpus_are_scarce: 'true'` parameter to reserve GPUs exclusively for services that opt-in to consume GPUs via the [Mesos `GPU_RESOURCES` framework capability]. Here is an example of setting this capability in a C++-based service.
```

Comment:
Do you think we need to include the full C++ GPU resources example? 
- I think we could simply point to the Mesos doc since this snippet by itself is unlikely enough information to set this capability, (they'll probably need the whole Mesos doc topic to figure out).
- And this is more a framework developer consideration, right? This content is more geared toward someone starting a DC/OS cluster with GPUs, for an existing GPU-enabled service.